etcdctlv3/command/util.go

```diff
@@ -47,3 +52,9 @@ func argify(s string) []string {
 	r := regexp.MustCompile("'.+'|\".+\"|\\S+")
 	return r.FindAllString(s, -1)
 }
+
+func commandCtx(cmd *cobra.Command) (context.Context, context.CancelFunc) {
+	var timeOut time.Duration
+	cmd.Flags().DurationVar(&timeOut, "timeout", defaultTimeOut, "Timeout for the request")
```

Comment:
i do not think this works. 

also probably we should call this `--command-timeout`, "Timeout for short running command (excluding dial timeout)"